x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/legacy.ts

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
It used now to run the new platform plugin under the legacy platform - will be removed after migration, because Plugin start and setup will be run by the NP itself